environment/docker-compose-arch.yml

```diff
@@ -0,0 +1,22 @@
+version: "3.2"
```

Comment:
Wonder if this should live in the docs folder as it's not part of the environment for running Core?

Reply:
👍  Good spot. Leftover from initial spike.